File: January 11, 2026.js
function golfScore(par, strokes) {

  if (strokes == 1) {
    return "Hole in one!";
  } else if (par - strokes >= 2) {
    return "Eagle";
  } else if (par - strokes == 1) {
    return "Birdie";
  } else if (par == strokes) {
    return "Par";
  } else if (strokes - par == 1) {
    return "Bogey";
  }

  return "Double bogey";
}

File: January 11, 2026.py
def golf_score(par, strokes):

    if strokes == 1:
        return "Hole in one!"
    elif par - strokes >= 2:
        return "Eagle"
    elif par - strokes == 1:
        return "Birdie"
    elif par == strokes:
        return "Par"
    elif strokes - par == 1:
        return "Bogey"

    return "Double bogey"